siliconcompiler/tools/vpr/route.py

```diff
@@ -0,0 +1,84 @@
+import os
+import re
+import shutil
+
+from siliconcompiler.tools.vpr import vpr
+
+
+def setup(chip):
+    '''
+    Perform automated place and route with VPR
+    '''
+
+    tool = 'vpr'
+    step = chip.get('arg', 'step')
+    index = chip.get('arg', 'index')
+    task = chip._get_task(step, index)
+
+    chip.set('tool', tool, 'exe', tool, clobber=False)
+    chip.set('tool', tool, 'version', '0.0', clobber=False)
```

Comment:
don't set if we don't have a way to check versions
If there is a method to check the VPR version, that should be implemented.